src/index.html

```diff
@@ -9,6 +9,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <main class = "main">
+      <header class = "header">
+        <a href="index.html">
+          <img src="./images/logo.png" 
+          alt="MOYO logotype" 
+          class = "MOYOimg">
+        </a>
+        <nav class = "nav">
+          <ul class = "nav_list">
+            <li class = "nav_item">
+              <a href="#apple" class = "nav_links is-active">Apple</a>
+            </li>
```

Comment:
Delete redundant spaces for all attributes
```suggestion
        <nav class="nav">
          <ul class="nav_list">
            <li class="nav_item">
              <a href="#apple" class="nav_links is-active">Apple</a>
            </li>
```